metro-ai-suite/image-based-video-search/compose.yml

```diff
@@ -66,6 +66,10 @@ services:
       context: src/streaming-pipeline
       dockerfile: Dockerfile
     container_name: ibvs-streaming
+    environment:
+      https_proxy: "${https_proxy}"
+      http_proxy: "${http_proxy}"
+      no_proxy: "${no_proxy}"

```

Comment:
From the diagram, we can see that RTSP does not use HTTP, so this proxy variables should not affect comms with the rtsp-server at `rtsp://rtsp-server:8554/stream`
 
![image](https://github.com/user-attachments/assets/3aa0003f-e3e8-4f0f-8bd3-0617900a7397)

But it could affect it if we do API calls to the server, using `http://rtsp-server:<API PORT>`. 

So this is good for now

Reply:
The wget command requires proxy setup to download file.mp4.